AutomatedTesting/Passes/ClipmapGeneration.pass

```diff
@@ -0,0 +1,52 @@
+{
```

Comment:
Can/should this file live in Gems/Terrain/Passes or something?

Reply:
Besides moving all these source assets to the Terrain Gem folder, We need to reconsider the name of the ClipmapGeneration.pass, .cpp, etc. We don't know yet if We'll be able to consolidate the same shaders and cpp classes for both MacroTextures and DetailTextures. So, for now, I recommend to rename ClipmapGeneration.* to TerrainDetailTextureComputePass.*. I'm naming my files as TerrainMacroTextureComputePass.*

Reply:
Good points. I'll put them in the right place and rename them.